src/functions/runtimeConfigExport.ts

```diff
@@ -72,13 +74,17 @@ export function getProjectInfos(options: {
  * Fetch and fill in runtime config for each project.
  */
 export async function hydrateConfigs(pInfos: ProjectConfigInfo[]): Promise<void> {
-  for (const pInfo of pInfos) {
-    try {
-      pInfo.config = await functionsConfig.materializeAll(pInfo.projectId);
-    } catch (err) {
-      logger.debug(`Failed to fetch runtime config for project ${pInfo.projectId}: ${err.message}`);
-    }
-  }
+  await Promise.all(
+    pInfos.map(async (info) => {
```

Comment:
You're calling `Promise.all()` on an array of async functions. You need to actually invoke your lambda to get it to be an array of Promises.

Reply:
Used temp var to simplify this a bit.